### PR TITLE
CLI that works with latest Typed TimeNode

### PR DESCRIPTION
**Pull request by lsaether**

### CODE REVIEW

File: cli/eac.js [hunk 1]
```diff
@@ -436,54 +436,80 @@ Endowment: ${web3.fromWei(endowment.toString())}
...
+    // tx.then((receipt) => {
+    //   if (receipt.status != '0x1') {
+    //     spinner.fail(`Transaction was mined but failed. No transaction scheduled.`)
+    //     process.exit(1)
+    //   }
+    //   spinner.succeed(`Transaction successful! Hash: ${receipt.transactionHash}\n`)
+    // })
+    // .catch((err) => {
+    //   spinner.fail(err)
+    // })
```
Comment (Bagaric) [inline cli/eac.js]: Do we need this commented-out part?

File: package.json [hunk 2]
```diff
@@ -32,8 +32,8 @@
     "commander": "2.15.1",
     "console.table": "0.10.0",
     "dotenv": "5.0.1",
-    "eac.js-client": "3.0.2",
-    "eac.js-lib": "2.0.2",
+    "eac.js-client": "file:../eac.js-client",
```
Comment (Bagaric) [inline package.json]: We should replace this before merging.

File: cli/eac.js [hunk 3]
```diff
@@ -14,15 +14,15 @@ const loki = require("lokijs")
 
 // CLI Imports
 const { Analytics } = require("./analytics")
-const Logger = require("./logger")
+// const Logger = require("./logger")
```
Comment (Bagaric) [inline cli/eac.js]: The logger is no longer used?

File: cli/eac.js [hunk 4]
```diff
@@ -369,37 +275,44 @@ const main = async (_) => {
       console.log("  error: must be running a localnode on the Ropsten or Kovan networks")
       process.exit(1)
     }
-    if (!await eac.Util.checkForUnlockedAccount()) process.exit(1)
+
+    if (!program.wallet || !program.password) throw new Error('Use a wallet');
+
+    const wallet = new Wallet(web3);
+    wallet.decrypt(JSON.parse(fs.readFileSync(program.wallet[0])), program.password);
```
Comment (Bagaric) [inline cli/eac.js]: I would detangle this line for better readability.

Comment (lsaether) [reply cli/eac.js]: Can add this in to a later PR.

Comment (Bagaric) [reply cli/eac.js]: 2018: // Temporary: Detangle this in the next PR
2122: // Don't touch, magic

**Pull request merged**